packages/account/src/providers/provider.test.ts

```diff
@@ -882,8 +882,8 @@ describe('Provider', () => {
 
     expect(consoleWarnSpy).toHaveBeenCalledOnce();
     expect(consoleWarnSpy).toHaveBeenCalledWith(
-      `The Fuel Node that you are trying to connect to is using fuel-core version ${FUEL_CORE}, 
-which is not supported by the version of the TS SDK that you are using. 
+      `The Fuel Node that you are trying to connect to is using fuel-core version ${FUEL_CORE},
```

Comment:
Are these formatting changes relevant?

Reply:
@maschad I don't even remember committing this.

This was fixed automatically by VSCode after making changes and saving them at `provider.ts`

I remember to avoid committing this but it got pushed anyway.

Therefore I had to adjust the tests

Reply:
As long as you configure your tooling correctly, avoiding committing lint fixes that may have slipped through a previous PR is tricky.

Reply:
@nedsalk I believe you were the last to modify these files. It's probably good to double-check to see if something like "lint on save" is working as expected. Otherwise, extraneous whitespace wouldn't go through.

Reply:
Adding `  "files.trimTrailingWhitespace": true,` to the [settings.json](https://github.com/FuelLabs/fuels-ts/blob/master/.vscode/settings.json#L7) may catch these slips. I'll resolve this comment since it's not relevant to the PR.